DuckDuckGo/TextZoomCoordinator.swift

```diff
@@ -51,7 +51,7 @@ protocol TextZoomCoordinating {
     func onTextZoomChange(applyToWebView webView: WKWebView)
 
     /// Shows a text zoom editor for the current webview. Does nothing if the feature is disabled.
-    func showTextZoomEditor(inController controller: UIViewController, forWebView webView: WKWebView)
+    func showTextZoomEditor(inController controller: UIViewController, forWebView webView: WKWebView) async
```

Comment:
What is the reason for that change?

Reply:
fix warning about concurrency and isolation that would be an error in swift 6